alibi_detect/saving/tests/test_saving.py

```diff
@@ -1152,6 +1176,8 @@ def test_save_preprocess_nlp(data, preprocess_fn, tmp_path, backend):
     assert cfg_preprocess['src'] == '@cd.' + registry_str + '.preprocess.preprocess_drift'
     assert cfg_preprocess['embedding']['src'] == 'preprocess_fn/embedding'
     assert cfg_preprocess['tokenizer']['src'] == 'preprocess_fn/tokenizer'
+    assert tmp_path.joinpath(cfg_preprocess['preprocess_batch_fn']).is_file()
+    assert cfg_preprocess['preprocess_batch_fn'] == 'preprocess_fn/preprocess_batch_fn.dill'
```

Comment:
This tests that the `preprocess_batch_fn` is serialized to disk, and has the correct name. Implicitly tests bug fix 1...